### PR TITLE
Added script for running release tasks

### PR DESCRIPTION
**Pull request by LilleJulaften**

### CODE REVIEW

File: Procfile [hunk 1]
```diff
@@ -1,3 +1,2 @@
-release: bundle exec rails db:migrate
-release: bundle exec rails csv:load
+release: bash ./release-tasks.sh
```
Comment (gingermusketeer) [inline Procfile]: I don't think the `bash` part is needed here as there is a hash bang in the file which determines how the file should be run.

File: bin/bash/release-tasks.sh [hunk 2]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```
Comment (gingermusketeer) [inline bin/bash/release-tasks.sh]: This file has a `.sh` extension but we are using `bash` to run it. Perhaps we should drop the `.sh` extension?

File: Procfile [hunk 3]
```diff
@@ -1,3 +1,2 @@
-release: bundle exec rails db:migrate
-release: bundle exec rails csv:load
+release: release-tasks
```
Comment (gingermusketeer) [inline Procfile:1]: Does this need to be `bin/bash/release-tasks` or does running `release-tasks` from the root of the repo work for you?

Comment (LilleJulaften) [reply Procfile:1]: We can leave it at root. No worries. Let's try it

**Review by gingermusketeer — APPROVED** [review]

**Pull request merged**